tests/beaker/test_run_scripts/integration_tests.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Init
+SCRIPT_PATH=$(pwd)
+BASENAME_CMD="basename ${SCRIPT_PATH}"
+SCRIPT_BASE_PATH=`eval ${BASENAME_CMD}`
+
+if [ $SCRIPT_BASE_PATH = "test_run_scripts" ]; then
+  cd ../
+fi
+
+# Work-around for RE-5005
+export SSL_CERT_FILE=/usr/local/etc/openssl/cert.pem
```

Comment:
This always causes problems for me. You may want to wrap this in a if statement to test for the path first.